tools/test-proxy/Azure.Sdk.Tools.TestProxy/CommandOptions/Config/ConfigShowOptions.cs

```diff
@@ -0,0 +1,34 @@
+using System.CommandLine;
```

Comment:
For the show and other commands, if possible it would be nice to set `.Required = true` on the options are required via System.CommandLine. Otherwise the user gets a less helpful error when missing a flag:

```
↘ ↘ ↘ dotnet run --framework net6.0 --project ./Azure.Sdk.Tools.TestProxy -- config show -f ~/sdk/azure-sdk-for-go
Running proxy version is Azure.Sdk.Tools.TestProxy 1.0.0-dev.20230421.1
git --version
Unhandled exception: System.ArgumentNullException: Path cannot be null. (Parameter 'path')
   at System.IO.Strategies.FileStreamHelpers.ValidateArguments(String path, FileMode mode, FileAccess access, FileShare share, Int32 bufferSize, FileOptions options, Int64 preallocationSize)
   at System.IO.File.OpenRead(String path)
   at Azure.Sdk.Tools.TestProxy.Startup.Run(Object commandObj) in /home/ben/sdk/azure-sdk-tools/tools/test-proxy/Azure.Sdk.Tools.TestProxy/Startup.cs:line 87
   at Azure.Sdk.Tools.TestProxy.CommandOptions.OptionsGenerator.<>c__DisplayClass0_0.<<GenerateCommandLineOptions>b__12>d.MoveNext() in /home/ben/sdk/azure-sdk-tools/tools/test-proxy/Azure.Sdk.Tools.TestProxy/CommandOptions/OptionsGenerator.cs:line 110
```

Reply:
Definitely makes sense.

Reply:
Huh. This is odd. It already is. I must be missing another config item.